frontend/docker-compose-e2e-tests.yml

```diff
@@ -16,6 +16,13 @@ services:
     build:
       context: ../
       dockerfile: Dockerfile
+      x-bake:
+        cache-from:
+          - type=gha,scope=image-e2e-unified-api
+        cache-to:
+          - type=gha,mode=max,scope=image-e2e-unified-api
+        tags:
+          - frontend_flagsmith-api
```

Comment:
I think this will make this docker-compose file unusable locally, right? Should we actually create a brand new docker-compose file and store it in the `.github` directory or something?

Reply:
Nope, I tested all this changes locally, but we should use `docker compose` instead of `docker-compose` since I am not sure if the latter accepts this file format.